rslearn/log_utils.py

```diff
@@ -0,0 +1,22 @@
+"""Logging utilities."""
+
+import logging
+import sys
+
+LOG_FORMAT = "format=%(asctime)s loglevel=%(levelname)-6s logger=%(name)s %(funcName)s() L%(lineno)-4d %(message)s"
+# DETAILED_LOG_FORMAT = "format=%(asctime)s loglevel=%(levelname)-6s logger=%(name)s %(funcName)s() L%(lineno)-4d %(message)s   call_trace=%(pathname)s L%(lineno)-4d"  # noqa
+
+
+def get_logger(name: str) -> logging.Logger:
```

Comment:
This seems to break the RSLEARN_LOGLEVEL environment variable. Instead of always setting to DEBUG, can you set it based on the environment variable like it does currently?